tests/jballerina-unit-test/src/test/resources/test-src/statements/assign/var-negative.bal

```diff
@@ -73,3 +73,8 @@ function test10 () {
     var x = <Foo> bar;
     string result1 = x + ex;
 }
+
+function test11 () {
```

Comment:
Shall we add more test cases? Eg- https://github.com/ballerina-platform/ballerina-lang/issues/10865#issuecomment-440537102

Reply:
Added more test cases.